python/tests/bathclamp_vclamp_step_response.py

```diff
@@ -228,12 +229,11 @@ def set_cmd_cc(dc_nums, cmd_val=0x1d00, cc_scale=0.351, cc_delay=0, fc=4.8e3, st
 ads.hw_reset(val=False)
 ads.set_host_mode()
 ads.setup()
-ads.set_range(ads_voltage_range) # TODO: make an ads.current_voltage_range a property of the ADS so we always know it
+ads.set_range(ads_voltage_range) 
 ads.set_lpf(376)
-# 4B - clear sine wave set by the Slow DAC
-ads_sequencer_setup = [('0', '0'), ('1', '1')]
+ads_sequencer_setup = [('0', '0'), ('1', '1'), ('2', '2')]
 codes = ads.setup_sequencer(chan_list=ads_sequencer_setup)
-ads.write_reg_bridge(clk_div=200) # 1 MSPS rate (do not use default value which is 200 ksps)
```

Comment:
Is this change correct? It looks like [`write_reg_bridge`](https://github.com/Ajstros/pyripherals/blob/main/python/src/pyripherals/peripherals/ADS8686.py#L102) defaults to `clk_div=1000`, which from the comment seems like it caused issues before.